content/references/rippled-api/ledger-data-formats/ledger-object-types/ripplestate.md

```diff
@@ -1,9 +1,14 @@
 # RippleState
```

Comment:
Comments on this file should go to #553, not here. (This file will leave the diff after I merge that PR and rebase.)